Documentation/BuildInstructions.md

```diff
@@ -3,33 +3,41 @@
 ### Prerequisites
 
 #### Linux prerequisites
-Make sure you have all the dependencies installed:
+Make sure you have all the dependencies installed (`ninja` is optional, but is faster in practice):
 
 **Debian / Ubuntu**
 ```bash
-sudo apt install build-essential cmake curl libmpfr-dev libmpc-dev libgmp-dev e2fsprogs qemu-system-i386 qemu-utils
+sudo apt install build-essential cmake curl libmpfr-dev libmpc-dev libgmp-dev e2fsprogs ninja qemu-system-i386 qemu-utils
 ```
 
 **Fedora**
 ```bash
 sudo dnf install curl cmake mpfr-devel libmpc-devel gmp-devel e2fsprogs @"C Development Tools and Libraries" @Virtualization
 ```
 
+<!-- FIXME: Add ninja -->
+
 **openSUSE**
 ```bash
 sudo zypper install curl cmake mpfr-devel mpc-devel gmp-devel e2fsprogs patch qemu-x86 qemu-audio-pa gcc gcc-c++ patterns-devel-C-C++-devel_C_C++
 ```
 
+<!-- FIXME: Add ninja -->
+
 **Arch Linux / Manjaro**
 ```bash
 sudo pacman -S --needed base-devel cmake curl mpfr libmpc gmp e2fsprogs qemu qemu-arch-extra
 ```
```

Comment:
For Archlinux: (Not sure what Manjaro is but it seems to be based on Archlinux, so this is probably okay.)

~~~suggestion
**Arch Linux / Manjaro**
```bash
sudo pacman -S --needed base-devel cmake curl mpfr libmpc gmp e2fsprogs qemu qemu-arch-extra ninja
```
~~~